spec.html

```diff
@@ -15898,7 +15898,7 @@ <h1>[[OwnPropertyKeys]] ( ): either a normal completion containing a List of pro
             The returned List contains no duplicate entries.
           </li>
           <li>
-            The Type of each result List element is either String or Symbol.
+            Each element of the result List element is a property key.
```

Comment:
```suggestion
            Each element of the returned List is a property key.
```

For symmetry with above step (plus grammar error).

Reply:
(Yeah, but on the other hand, the subsequent points refer to "the result List".)